src/pages/MyPartyDetailPage.jsx

```diff
@@ -1,47 +1,51 @@
-import { Box } from '@mui/system';
 import { useCallback, useState } from 'react';
-import PartyShareAccount from 'components/MyParty/PartyShareAccount';
-import { PageContainer, PageContents, PageHeader } from 'components/Common';
-import PartyTitle from 'components/PartyTitle';
-import { PARTY_DETAIL_DUMMY } from 'constants/mockData/parttDetailDummy';
-import PartyMemberList from 'components/Common/PartyMemberList';
-import RuleContainer from 'components/Common/Rule';
-import { Divider, Typography } from '@mui/material';
 import { useParams } from 'react-router-dom';
+import { Box, Chip, Divider, Typography } from '@mui/material';
 import { AddCircleOutline, RemoveCircleOutline } from '@mui/icons-material';
+import { getMyPartyById, getSharedAccountInfo } from 'utils/api';
+import useAsync from 'hooks/useAsync';
+import { useAuthState } from 'contexts/authContext';
 import { priceToString } from 'utils/priceToString';
-import CardTemplate from 'components/Common/CardTemplate';
-
-const MY_USER_ID = 2;
+import PartyTitle from 'components/PartyTitle';
+import {
+  PageContainer,
+  PageContents,
+  PageHeader,
+  PartyMemberList,
+  CardTemplate,
+} from 'components/Common';
+import RuleContainer from 'components/Common/Rule';
+import PartyShareAccount from 'components/MyParty/PartyShareAccount';
 
 const MyPartyDetailPage = () => {
+  const { userId } = useAuthState();
+  const MY_USER_ID = userId;
```

Comment:
```suggestion
  const { userId: myUserId } = useAuthState();
```
구조분해할당 후 재할당해줘도 좋을것 같아요!

Reply:
그렇게 하면 되겠네요! 역시!